refugee/refugeeController.js

```diff
@@ -89,3 +122,18 @@ module.exports.findRefugees = function (req, res) {
     res.status(200).json(refugees);
   });
 };
+
+module.exports.updateResume = function (req, res) {
+	
+	var myresume = req.body.params.refugee;
+	var myquery = { _id: req.body.params.refugee._id };
+	delete myresume._id;
+	Refugee.updateOne(myquery, myresume, function(err, result) {
```

Comment:
Same here. Here the risk is if you update a document with some data, you have to be sure that you only update fields that you are working with. For instance if `name` is not updated but you still set `name` field as `undefined`, it will overwrite name field. Could you please verify this situation?